utils/user.js

```diff
@@ -53,8 +53,9 @@ class User {
    * @returns {{email: string, password: string}}
    */
   get googleCreds() {
-    if (process.env[this.envGoogleEmail] === '' || process.env[this.envGooglePassword] === '') {
-      throw Error(GOOGLE_CREDS_MISSING_ERROR + this.username);
+    if (!process.env[this.envGoogleEmail] || process.env[this.envGoogleEmail] === '' || !process.env[this.envGooglePassword] || process.env[this.envGooglePassword] === '') {
```

Comment:
why are the additional `!process.env[this.envGoogleEmail]` necessary? doesn't `''` evaluate to false? or am I thinking too much like Python..